src/sinks/util/adaptive_concurrency/tests.rs

```diff
@@ -82,16 +82,17 @@ impl LimitParams {
     }
 
     fn scale(&self, level: usize) -> f64 {
-        (level - 1) as f64 * self.scale
-            + self
-                .knee_start
+        ((level - 1) as f64).mul_add(
+            self.scale,
+            self.knee_start
```

Comment:
Really? `mul_add` is less cognitively difficult than standard infix operators?

Reply:
I think it's more that it rounds less:

```
Fused multiply-add. Computes (self * a) + b with only one rounding error, yielding a more accurate result than an unfused multiply-add.
```

Reply:
Seems about the same to me. It's something that came in with clippy fix.